resources/compass/charts/provisioner/values.yaml

```diff
@@ -1,10 +1,17 @@
 deployment:
   image:
     pullPolicy: IfNotPresent
-
   securityContext: # Set on container level
     runAsUser: 2000
     allowPrivilegeEscalation: false
+  liveness:
+    initialDelaySeconds: 30
+    timeoutSeconds: 1
+    periodSeconds: 10
+  readiness:
+    initialDelaySeconds: 5
+    timeoutSeconds: 1
+    periodSeconds: 2
```

Comment:
I think we could, but some `livenessProbe` `initialDelaySeconds` attributes might need specific values/fine-tuning later based on in-practice experience. (To avoid premature restart of containers for which initialization takes long.)
Should I keep all attributes in `.global` except the `livenessProbe.initialDelaySeconds` ?

Reply:
Hmm, that's sad - I thought we have the same values everywhere. Can we have like `initialDelaySeconds: 30` for all components?

If you think it's not good to have such high number of delay, then yes, please keep `initialDelaySeconds` per component, and the rest we could append from values. Thanks!

Reply:
For director it was already configured for 50s. I haven't checked but maybe it can take really that much amount of time for director to initialize and become ready?
If not, I'll just move `initialDelaySeconds` also to global with `30`